website-team/proposals/zola.md

```diff
@@ -0,0 +1,43 @@
+- **Asset collocation —** Zola by default allows you to host static content from the same directory as the source markdown, this removes some of the absolute URLs that required on the blog because the static assets directory is seperate.
+
+
+## URL Scheme Changes
+
+While Zola does provide the functionality we need it does not map directly to our current URL scheme, and merging the three websites will require moving the blog and thanks to be available under the `www.rust-lang.org` host. The blog currently has the following URL scheme.
+
+**Current Scheme**
+
+    https://blog.rust-lang.org/{YYYY}/{MM}/{DD}/{title}.html
+    https://blog.rust-lang.org/inside-rust/{YYYY}/{MM}/{DD}/{title}.html
+
+This date based URL scheme is not supported in Zola and the Zola maintainers have said they are not interested in adding this date scheme. Additionally we have had issues with the date based URL scheme, as sometimes a blog post is merged with the wrong date, and then updating the link will break any links to the previous posts, such as on social media or in RSS readers. For these reasons there is motivation to also take the time to use merging onto `www.rust-lang.org` to also revisit this URL scheme and then make the switch once.
+
+Instead of following a date based scheme, we should try to replicate forge's layout of grouping by team for blog. For now this layout would **not** be reflected on the front end of blog itself, only in the internal layout and in the URL scheme. The blog's current flat list of articles grouped by year will remain unchanged.
+
+**Proposed Scheme**
+
+    https://www.rust-lang.org/blog/{topic}/{title}.html
+    https://www.rust-lang.org/blog/inside-rust/{team}/{title}.html
+    # Examples
+    https://www.rust-lang.org/blog/releases/1.44.0.html
+    https://www.rust-lang.org/blog/docs-rs/choose-your-build-target.html
+    https://www.rust-lang.org/blog/inside-rust/governance-wg/meetings/2020-04-05.html
+## Migrating to Zola
+
+Luckily the website being divided into three separate apps actually makes migrating to a single component easier, as we can by switching only a single component and slowly integrate the other two without requiring that maintenance work be stopped or blocked on the transition. The best candidate to start with is the Rust blog, the website currently has fluent localisation which is not currently supported by Zola (though it is being actively worked on), and "Thanks" is both pretty bespoke in terms of functionality and is the least popular of the three. Switching the blog allows us to test and iterate on Zola with a sizeable portion of our audience, while maintaining the current level of functionality.
```

Comment:
I'd start with thanks instead, as it's the simplest and there isn't much impact if something breaks.